libs/feeds_processing/src/utils.rs

```diff
@@ -605,3 +370,161 @@ pub async fn validate(
 
     Ok(())
 }
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+
```

Comment:
I'd like to see a test for the case where a reporter has reported on a subset of the feeds in the proposed batch update. In that case the reporter should reject the whole batch, as the batch corresponds to tx call data, and we can't submit it partially.

Reply:
Addressed in [PR-1153](https://github.com/blocksense-network/blocksense/pull/1153)